lifeapp/src/components/Form.js

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import Todo from "./Todo"
+
+
+class Form extends React.Component {
+constructor(props){
+    super(props)
+    this.state = {value: "", name: [], submit: false}
+
+}
+
+handler = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+    // debugger
+    // console.log(this.state.name)
+}
+
+todo = (e) =>{
+    e.preventDefault();    
+    this.setState({submit: true})
+    this.onAddItem();
+    this.notify(this.state.value)
+    // debugger
+};
+
+onAddItem = () => {
+    
+    this.setState(state => {
+      const name = state.name.concat(state.value);
```

Comment:
At this point you want to send back this value to the the state on the parent component